app/flatpak-builtins-utils.h

```diff
@@ -30,6 +30,16 @@
 /* Appstream data expires after a day */
 #define FLATPAK_APPSTREAM_TTL 86400
 
+typedef struct RemoteDirPair
+{
+  gchar              *remote_name;
```

Comment:
I think we are generally using g-less types in flatpak when possible, so char *

Reply:
I just copied this from elsewhere but if we want to enforce that norm we could do a replace-all across the codebase